openfisca_country_template/parameters/general/age_of_retirement.yaml

```diff
@@ -1,5 +1,5 @@
 # A parameter can contain utf-8 characters.
-description: Age of retirement (in years). Âge de la retraite (en années).
+description: Age of retirement (in years). In French âge de la retraite (en années).
```

Comment:
Maybe we could add ticks, quotes or parenteses around the French translation ?
And I'm wondering if `retirement age` woundn't be better than Age of retirement ?

Reply:
The simple fact that we use French to demonstrate UTF-8 is deficient. If we are to spend more time on that I'd rather have us find a way to bring in UTF-8 descriptions that make sense in English.
For me, that's already an improvement over the previous phrasing and I don't feel like doing a lot more.

Reply:
> And I'm wondering if retirement age woundn't be better than Age of retirement ?

Seems to me `retirement age` is the correct spelling, at least according to [Wikipedia](https://en.wikipedia.org/wiki/Retirement_age)

Reply:
> The simple fact that we use French to demonstrate UTF-8 is deficient.

Sure. I propose though we keep consistency with https://github.com/openfisca/country-template/pull/34/files#diff-e12bf04429356106f9750727cc8aea46R57, and use _In French: âge de la retraite (en années)._